framework/include/partitioner/MoosePartitioner.h

```diff
@@ -35,6 +35,7 @@ InputParameters validParams<MoosePartitioner>();
  * Base class for MOOSE partitioner
  */
 class MoosePartitioner :
+  public Partitioner,
```

Comment:
You will also need to add a corresponding line in the initialization list (.C file) to construct this new base class, right?